examples/covid/Covid Using Graphs in Jupyter Notebook.ipynb

```diff
@@ -0,0 +1,444 @@
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "983ec368",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%local \n",
+    "import plotly.express \n",
+    "plotly.express.bar(total_cases, x='province', y='count(1)', color='gender', title='Cases per Gender')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0c746b6",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    By looking through the data you can see that Quebec has a large amount of people who were classified as undefined. This is probably again due to Quebec's strict privacy laws.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d65f66df",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "The last dataset that we are importing is daily cases for the four provinces.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8b54ceff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%import_dataset covid19.canada.daily-cases --alias daily_cases"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44ff2b2a",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Now test again to see if the data was succcesfully installed for this file.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6884e659",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql -o daily_cases\n",
+    "select * from daily_cases"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "083fecc2",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "The last step is to create a line plot graph to compare the different amount of cases per day by province.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7169812b",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%%local\n",
+    "import plotly.express\n",
+    "plotly.express.line(daily_cases, x=\"reported_date\" , y=\"total_daily\", color=\"province\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bd2a6b0",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "As seen in the graph above, there are two significant spikes in Quebec's COVID-19 data from late December 2020 and early January 2021. These spikes likely don't reflect an actual surge in cases but rather a delay in data entry due to the holiday weekend. In Kamu, we use watermarks, which are markers that indicate when we have probably received all relevant data for a particular period. Given the potential for delays like these, the watermark can be adjusted to account for late-arriving data, ensuring that all reports are accurately included in the analysis.\n",
```

Comment:
Let's also link to the upstream issue where other people point out the same problem with the dataset:
https://github.com/ccodwg/Covid19Canada/issues/44

Reply:
Is it correct what I did with the commit linking to it?

Reply:
I actually meant adding the link in the notebook itself, to reference people who are interested in details of that issue.